src/pages/contribute/index.tsx

```diff
@@ -0,0 +1,187 @@
+                contributions to the project.
+              </Text>
+
+              <Flex justifyContent="center" wrap="wrap">
+                <Link
+                  as={NextLink}
+                  href="/contribute/getting-started"
+                  isExternal={false}
+                >
+                  <Button size="large" variation="primary">
+                    Get Started Contributing to Amplify
+                  </Button>
+                </Link>
+
+                <Link
+                  as={NextLink}
+                  href="https://github.com/aws-amplify/"
+                  isExternal={true}
+                >
+                  <Button size="large">
+                    <Text>
+                      Explore the repos <FiExternalLink />
+                    </Text>
+                  </Button>
+                </Link>
+              </Flex>
+            </Flex>
+
+            <HowItWorks />
+
+            <Flex
+              direction="column"
+              maxWidth="1024px"
+              width={{ base: '90%', large: '60%' }}
+              margin="0 auto"
+              gap="2em"
+            >
+              <View>
+                <Heading level={2}>Start contributing!</Heading>
+              </View>
+              <Text fontSize={'large'} color="font.secondary">
+                Thank you for your interest in contributing! Whether it's a bug
+                report, new feature, correction, or additional documentation, we
+                greatly value feedback and contributions from our community.
+                Check out the issues and discussions in the Amplify projects
+                below to get started!
+              </Text>
+            </Flex>
+
+            <Flex
+              direction="row"
+              justifyContent="center"
+              alignItems="top"
+              gap="5em"
+              wrap={'wrap'}
+            >
+              <Card
+                padding={'3rem'}
+                variation="elevated"
+                width={{ base: '90%', large: '40%' }}
+              >
+                <Issues
+                  title={'Amplify JS'}
+                  issues={JsIssues}
+                  repo={'amplify-js'}
+                  description="A declarative JavaScript library for application development using cloud services."
+                />
+              </Card>
+              <Card
+                padding={'3rem'}
+                variation="elevated"
+                width={{ base: '90%', large: '40%' }}
+              >
+                <Issues
+                  title={'Amplify CLI'}
+                  issues={CLIissues}
+                  repo={'amplify-cli'}
+                  description="The AWS Amplify CLI is a toolchain for simplifying serverless web and mobile development."
+                />
+              </Card>
+            </Flex>
+
+            <QuickstartResources />
+            {/* <Contributors /> */}
```

Comment:
Are we not including this component anymore?

Reply:
We'll open an issue for this and follow up.